projects/control-service/projects/model/apidefs/datajob-api/api.yaml

```diff
@@ -1249,6 +1300,14 @@ components:
         redshift-password: bar
         last-execution-time: 1585303338
 
+    DataJobSecrets:
+      description: Secrets of a Data Job
```

Comment:
If I am a user looking at the Swagger API I may wonder 

* What is "secrets" ? What is the difference between secrets and properties. When to use one vs the other. 
* 
Some more explanation might be beneficial to future users.